Shared/Samples/Identify KML features/IdentifyKMLFeaturesView.swift

```diff
@@ -15,6 +15,7 @@
 import ArcGIS
 import SwiftUI
 
+@MainActor
```

Comment:
Instead of annotating the whole view as MainActor, I think it is worth to refactor the view to not passing the proxy around, to avoid the change. No need to revert, will fix in https://github.com/Esri/arcgis-maps-sdk-swift-samples/issues/455

Reply:
Annotating the view as `MainActor` is a good solution because every SwiftUI view will be constrained to the main actor when building with the iOS 18 SDK.